### PR TITLE
Hosted content - about information

### PR DESCRIPTION
**Pull request by uplne**
## What does this change?

Small popup to give more information about this product

screen shot 2016-05-16 at 14 29 34

### CODE REVIEW

**Comment by Calanthe** [comment]
We have sans serif font in the standard "Glabs' About" text:
screen shot 2016-05-16 at 15 06 55
 
Why is it serif on the hosted page?

File: commercial/app/views/static/guardianHostedPage.scala.html [hunk 1]
```diff
@@ -5,6 +5,13 @@
 @main(model.HostedPage(metaData)) { } {
     <div class="hosted__header">
         <div class="hosted__headerwrap">
+            <div class="hosted__label">
+                <button class="u-button-reset hosted__labelbtn popup__toggle" data-toggle="hosted__popup">About @fragments.inlineSvg("arrow-down", "icon", List("hosted__labelicon"))</button>
+                <div class="popup popup--default is-off hosted__popup">
+                    <h3 class="hosted__labektitle">Advertiser content</h3>
```
Comment (Calanthe) [inline commercial/app/views/static/guardianHostedPage.scala.html]: What does `labek` mean?

Also, I would use `hosted__label-btn`, `hosted__label-text` classes.

File: static/src/javascripts/bootstraps/enhanced/common.js [hunk 2]
```diff
@@ -141,13 +141,14 @@ define([
...
                         && config.page.contentType !== 'Crossword'
+                        && config.page.contentType !== 'Hosted'
                         && !config.page.isImmersive
                         && !config.page.isUsMinute
                         && !config.page.isAdvertisementFeature
                         )
                 ) {
-                    stickyAdBanner.initialise();
-                    config.page.hasStickyAdBanner = true;
+                    //stickyAdBanner.initialise();
```
Comment (Calanthe) [inline static/src/javascripts/bootstraps/enhanced/common.js]: Why is it commented out?

**Comment by uplne** [comment]
Lol thanks for the comments @Calanthe I am closing this one as I shouldn't create PR form this yet. I will reopen after I will merge the previous PR.

**Pull request closed without merging**